spec/features/audit/lists/filter_spec.rb

```diff
@@ -153,6 +153,27 @@
     end
   end
 
+  context "filtering by title" do
+    scenario "the user enters a text in the search box and retrieves a filtered list" do
+      create :content_item, title: "some text"
+      create :content_item, title: "another text"
```

Comment:
Is there any way to make the search case insensitive? When I search for `Driving` and `driving` I get different sets of results.

Reply:
Address in e390f6c